cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://localhost:3002"
```

Comment:
I would add the security setting from start

Reply:
done. so then i guess it was a good decision to set baseUrl to localhost:3002?